functions/src/controllers/notifications.js

```diff
@@ -0,0 +1,56 @@
+const { OK } = require('http-status-codes');
+const { db } = require('../utils/firebase');
+const { tryCatchError, validationError } = require('../utils/errorHandler');
+const { successNoMessage, successNoData } = require('../utils/successHandler');
+
+class notifications {
+
+    /**
+	 * Get all notification, only authenticated users
+	 * @function
+	 * @param {object} req - request object
+	 * @param {object} res - response object
+	 * @return  {Object} result
+	 */
+    static async allNotifications(req, res) {
+        try {
+            const { userId } = req.user;
+            const data = await db.collection('notifications')
+                .where('userId', '==', userId).where('read', '==', false)
+                .orderBy('createdAt', 'desc').get();
+            const docs = data.docs, notifications = [];
```

Comment:
don't do this, put notifications on its own line with its own `const`

Reply:
Okay.